deposit/src/main/java/edu/unc/lib/deposit/transfer/TransferBinariesToStorageJob.java

```diff
@@ -94,6 +96,7 @@ public void runJob() {
         // All objects in deposit should have the same destination, so pull storage loc from deposit record
         try (BinaryTransferSession transferSession = getTransferSession(model)) {
             transferBinaries(depositBag, transferSession);
+            setTotalClicks(ingestedObjCount);
```

Comment:
Hmm, with this approach the total count will shrink if you resume the job. It may be easiest to switch the count to be the number of objects being processed rather than the number of files. In that case you should be able to precompute the total (I believe counting the total number of subjects in the model will get you this), and the counts will be in line with most of the other jobs